pkg/api/types.go

```diff
@@ -673,6 +673,9 @@ type GlusterfsVolumeSource struct {
 	// Required: EndpointsName is the endpoint name that details Glusterfs topology
```

Comment:
Is this Required now?  

You cannot really make this un-Required because clients might be looking for this field?